src/java.base/share/classes/java/time/chrono/HijrahChronology.java

```diff
@@ -1035,9 +1036,8 @@ private static void registerCustomChrono() {
         AccessController.doPrivileged(
             (PrivilegedAction<Void>)() -> {
                 if (Files.isDirectory(CONF_PATH)) {
-                    try {
-                        Files.list(CONF_PATH)
-                            .map(p -> p.getFileName().toString())
+                    try (Stream<Path> stream = Files.list(CONF_PATH)) {
+                            stream.map(p -> p.getFileName().toString())
```

Comment:
I can't tell if there is a tab here but it looks like you've changed this to indent by 8 instead of 4 spaces.